pages/index.js

```diff
@@ -83,7 +97,14 @@ export default function Home() {
             Experiencing an issue with this product or have you spotted an
             error?
           </p>
-          <button>Report a problem</button>
+          <ActionButton
+            text={"Report a problem"}
+            rounded
+            className={
+              "bg-white text-blue-600 border border-blue-600 w-auto hover:bg-gray-200"
+            }
```

Comment:
We pretty have 2 button styles so far, the blue and the alternate white with blue text. Can we make this the default for our use-case?